Source/WebCore/dom/TreeScope.h

```diff
@@ -71,6 +71,10 @@ class TreeScope {
     void ref() const;
     void deref() const;
 
+    // For CheckedPtr / CheckedRef use.
+    void incrementPtrCount() const;
+    void decrementPtrCount() const;
```

Comment:
A little surprised by the names of these functions. I guess I’m not all that familiar with `CheckedPtr` internals.

Reply:
Maybe we should rename to incrementCheckedPtrCount() / decrementCheckedPtrCount?

Or we could call them checkedPtrRef / checkedPtrDeref.